ibis/expr/operations.py

```diff
@@ -835,7 +835,7 @@ def output_type(self):
 
 class Arbitrary(Reduction):
     arg = Arg(rlz.column(rlz.any))
-    how = Arg(rlz.isin({'first', 'last', 'heavy'}), default='first')
+    how = Arg(rlz.isin({None, 'first', 'last', 'heavy'}), default=None)
```

Comment:
`None` should work since you have it as the default, i.e., you shouldn't have to include `None` in the `rlz.isin` call. Does it fail if you don't include it?

Reply:
mm I didn't know that, thanks. I will try that!